tool/score/fix_excludes.yaml

```diff
@@ -0,0 +1,17 @@
+# A list of rules to which there are no reasonable auto-fix
+# semantics.
+#
+# Rules in this list will be excluded from scoring.
+
+- avoid_as
+- avoid_catching_errors
+- avoid_classes_with_only_static_members
+- avoid_implementing_value_types
+- avoid_positional_boolean_parameters
+- avoid_returning_null_for_future
+- avoid_returning_this
+- avoid_slow_async_io
```

Comment:
We could have a fix to convert the async member to the sync member (and remove the `await` in the process). I don't know that it's worth doing, however.

Reply:
Doesn't sound like a bad proposal actually.  I'll add it back in.